.github/workflows/code_quality.yml

```
@@ -0,0 +1,33 @@
+name: Code Quality (rustfmt and clippy)
```

```suggestion
name: Code Quality
```

The UI shows the job names, so there's no need to repeat them here.

---

done